addons/mail/static/src/views/activity/activity_compiler.js

```diff
@@ -12,8 +12,17 @@ export class ActivityCompiler extends ViewCompiler {
         let compiled;
         if (!el.hasAttribute("widget")) {
             // fields without a specified widget are rendered as simple spans in activity records
-            compiled = createElement("div", { "class": "d-inline-block", "t-out": `record["${el.getAttribute("name")}"].value` });
```

Comment:
`d-inline-block` cannot just be removed as it is breaking other things (eg. on product). Only conditionally added when it's not conflicting with another display rule